rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_14.json

```diff
@@ -0,0 +1,1314 @@
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 1",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 11",
+                                                        "primary_airflow": 1000,
+                                                        "minimum_outdoor_airflow": 200
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
+                                                "id": "System 11",
+                                                "cooling_system": {
+                                                    "id": "CHW Coil 1",
+                                                    "cooling_system_type": "FLUID_LOOP",
+                                                    "chilled_water_loop": "Chilled Water Loop 1"
+                                                },
+                                                "heating_system": {
+                                                    "id": "Heating Coil 1",
+                                                    "heating_system_type": "ELECTRIC_RESISTANCE"
+                                                },
+                                                "fan_system": {
+                                                    "id": "VAV Fan System 1",
+                                                    "fan_control": "VARIABLE_SPEED_DRIVE",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 100,
+                                                            "design_airflow": 1000
+                                                        }
```

Comment:
I believe we also need the ```return_fans``` to make sure we don't fall into the Case 3. Could you add it?

Reply:
Added with 0 cfm and 0 kW

Reply:
So, if the ```fan_airflow``` equals 0 cfm, it doesn't count. Could you update the value here and the following rules relevant to this (e.g., (b), (c), etc.)?

Reply:
TCD 19-14-a is intended to test Case 2 of [the RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section19/Rule19-14.md) where there are no return or relief fans modeled in the proposed and 0 cfm for the return and relief in the baseline. 

TCD 19-14-b is a planned failed outcome of the same case where there are no return or relief fans modeled in the proposed, but greater than 0 cfm for the return in the baseline. 

The following TCDs should have greater than 0 cfm for the return fans.

Reply:
My apologies. I was confusing myself. Thank you for clarifying this!

Reply:
It turned out that my confusion was because I didn't pull your branch to see the latest updates.